backend/src/Request/Admin/StoreOwnerDuesFromCashOrders/CaptainAmountFromOrderCashFilterGetRequest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Request\Admin\StoreOwnerDuesFromCashOrders;
+
+class StoreOwnerDuesFromCashOrdersFilterGetRequest
+{
+   private int $storeId;
+
+   private string $fromDate;
+   
+   private string $toDate;
```

Comment:
If the filtering options are not required, then we have to set also null as a var type.